storybook/preview.js

```diff
@@ -19,6 +19,7 @@ import './style.scss';
 
 addDecorator( withA11y );
 addDecorator( withKnobs );
+
 configure(
```

Comment:
This should be part of `main.js` now, right?

Reply:
That's my guess as well. However, I wasn't sure how to do it given the conditional `require` we currently have. I'll give it a shot!